ygong/mosaic/trainingConfig.py

```diff
@@ -59,5 +59,10 @@ def toRunConfig(self, scalingConfig: ScalingConfig, wsfs: Optional[WSFSIntegrati
             compute=scalingConfig.toCompute,
             integrations=self.hacky_integrations,
             env_variables=env_variables,
+            scheduling={
+                'priority': scalingConfig.priority,
+                'preemptible': scalingConfig.preemptible,
+                'retry_on_system_failure': scalingConfig.retry_on_system_failure
+            },
```

Comment:
can you encapsulate this within  scalingConfig, so would be like `scheduling=scalingConfig.toScheduling()`